src/UNL/Peoplefinder/Driver/LDAP/StandardFilter.php

```diff
@@ -51,6 +51,11 @@ function __construct($inquery, $operator = '&', $wild = false)
             
             //put the query into an array of words
             $query = preg_split('/\s+/', $inquery, 4);
+            
+            //remove empty parts
+            $query = array_filter($query, function($value) {
+                return !empty($value);
```

Comment:
This is the default behavior of array_filter, without callback (keep in mind that `0` is empty, probably doesn't matter).